More loadout choices: uniforms, chest rigs, holsters & hairbrushes

Ephemeralis opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

This PR adds the following items to the loadout selection screen:

Misc. Under: black bunny suit, white bunny suit, latex catsuit, geisha suit, jabroni outfit
Uniform: CIN combat uniform (normally importable from cargo)
Belts: Shoulder holsters (normal holsters you get when making w/ leather), cowboy belt (special thigh holster + belt sprite that works like an ordinary holster, very aesthetic, highly recommend), all CIN surplus chest rigs (normally importable from cargo), expeditionary chest rig (found in expedition surplus crates, also orderable from cargo)
Pocket: hairbrush, comb

The general rule-of-thumb when adding items to the loadout is that they should be already be readily and trivially accessible from the get-go, and I believe all of the above satisfy this criteria. Most of them are either basic 1-step crafts with easily-obtainable items, importable/orderable from cargo, or available in public vendors.

## How This Contributes To The Nova Sector Roleplay Experience

Loadout items are a major customization point for players, especially since they can be easily altered. 90% of the items added above were from requests fielded on #main-talk over about 25 minutes.

## Proof of Testing

Super simple changes. It compiles, if anything breaks, CI will let us know.

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. You can read up on GBP and it's effects on PRs in the tgstation guides for contributors. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->

:cl:
add: The following items are now available in the loadout menu: bunny suit (black), bunny suit (white), latex catsuit, geisha suit, jabroni outfit, CIN combat uniform, shoulder holsters, cowboy belt, all the CIN surplus chest rigs, expeditionary chest rig, hairbrush, and combs. 
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Stalkeros2 commented:
Holsters aren't guns, they're holsters.

Nerev4r commented:
yo can you throw in the kahraman chest rig as well

Stalkeros2 commented on modular_nova/modules/loadouts/loadout_items/under/loadout_datum_under.dm:
```diff
@@ -138,6 +138,10 @@ GLOBAL_LIST_INIT(loadout_miscunders, generate_loadout_items(/datum/loadout_item/
 	item_path = /obj/item/clothing/under/colonial/nri_police
 	restricted_roles = list(JOB_SECURITY_OFFICER, JOB_DETECTIVE)
```
I am too lazy to do this myself but could you un-restrict the not-NRI:tm: police uniform please; it's not like anyone can't and doesn't buy it from Cargo anyways nor like the Sol uniforms in this list are restricted themselves.

Ephemeralis replied:
Done this!

honkpocket commented:
>the rigs can hold stuff like advanced engi tools

And?

Stalkeros2 commented:
Oh, right- Whistles, too, would be nice to see.

kittysmooch commented:
if we're making whistles common then for the love of god we need to make them quieter

Ephemeralis commented:
- Removed the role restrictions for the NRI imperial police uniform
- Added the imperial police cloak
- Added the frontier chest rig
- Added the basic synthetic repair kits to pocket loadout slots

Stalkeros2 commented:
> while you're here, could you add the NRI police armor and hat/helmet to loadout? sec restriction obviously (its just reskinned sec armor that looks nice, you can also rename it in the loadout to have it not be NRI)

They were in the loadout before ~~for a short period of time so people could get them more and check said stuff out~~ but got removed ~~when said short period ended~~ so- Yeah please do that.
